Tools/autotest/apmrover2.py

```diff
@@ -21,10 +21,7 @@
 testdir = os.path.dirname(os.path.realpath(__file__))
 
 # HOME = mavutil.location(-35.362938, 149.165085, 584, 270)
-HOME = mavutil.location(40.071374969556928,
-                        -105.22978898137808,
-                        1583.702759,
-                        246)
+HOME = mavutil.location(-35.362938, 149.165085, 584, 270)
```

Comment:
Can't really do this without consultation; tradition says we start there.

Reply:
reverted